accounts/addproofer.js

```diff
@@ -0,0 +1,29 @@
+/*global $*/
+
+function AddProofer() {
+  "use strict";
+
+  var that = this;
+  this.initForm = function() {
+    "use strict";
+
+    that.referrerOption = $('select[name=referrer]');
+    that.referrerDetails = $('#referrer_details');
+    that.setOtherDetails();
+    that.referrerOption.change(that.setOtherDetails);
+  };
+
+  this.setOtherDetails = function() {
+    "use string";
```

Comment:
Should this be `"use strict";` instead?

Sorry, just noticed this.

Reply:
/facepalm.